src/main/java/org/acra/builder/ReportExecutor.java

```diff
@@ -145,6 +146,7 @@ public void execute(@NonNull final ReportBuilder reportBuilder) {
                 || (config.resToastText() != 0 && (reportingInteractionMode == ReportingInteractionMode.NOTIFICATION || reportingInteractionMode == ReportingInteractionMode.DIALOG));
 
         final TimeHelper sentToastTimeMillis = new TimeHelper();
+        boolean displayingToast = false;
```

Comment:
That is a development artifact. I first tried to kill the process without triggering the kill on all other processes, which is why this was required. Now it is not and I'll remove it really quick.